src/components/pages/Theme/Theme.jsx

```diff
@@ -273,10 +294,20 @@ export const Theme = () => {
               placeholder='Select options'
             />
 
+            {inputTypes.map((type) => (
+              <Form.Field
+                className='formLabel'
+                control='input'
+                key={`input-${type}`}
+                label={type}
+                type={type}
+              />
+            ))}
+
             <Form.Field
               control={TextArea}
-              label='About'
-              placeholder='Tell us more about you...'
+              label='Textarea'
```

Comment:
Let's create a separate segment for `TextArea`.

Reply:
Should it be within the `Input` segment?